pandas/tests/arithmetic/test_numeric.py

```diff
@@ -712,6 +733,51 @@ def check(series, other):
         check(tser, 5)
 
 
+class TestUFuncCompat(object):
```

Comment:
let's think about putting these in a separate file (test_funcs)? (may want to make numeric a subdir, its already getting long).

Reply:
I'll give that some thought.  Preference is to do quick turnarounds with small PRs here since there are lots of bugs to fix.